### PR TITLE
Add rake jasmine tasks in docs

### PR DESCRIPTION
**Pull request by alex-ju**
In case other members of the team find this useful as a reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,20 @@ yarn install
 bundle exec rake
 ```
 
+#### Running JavaScript tests only
+
+Running [Jasmine][] tests in a web browser
```
Comment (kevindew) [inline README.md]: Might be worth phrasing this as "your desktop web browser"?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,20 @@ yarn install
...
+
+```
+rake jasmine
+```
+
+Running [Jasmine][] tests in CI
```
Comment (kevindew) [inline README.md]: Might be better to phrase this as run in console, as we use this task outside of CI too.

File: README.md [hunk 3]
```diff
@@ -47,6 +47,20 @@ yarn install
 bundle exec rake
 ```
 
+#### Running JavaScript tests only
+
+Running [Jasmine][] tests in a web browser
+
+```
+rake jasmine
```
Comment (kevindew) [inline README.md]: Probably prudent to prefix this with `bundle exec`

**Comment by alex-ju** [comment]
thanks for the comments @kevindew. updated as per comments.

**Pull request merged**